flatpak_builder_lint/appstream.py

```diff
@@ -12,8 +12,33 @@
 def validate(path: str) -> dict:
     if not os.path.isfile(path):
         raise FileNotFoundError("AppStream file not found")
+
+    overrides = {
+        "all-categories-ignored": "error",
+        "category-invalid": "error",
+        "cid-desktopapp-is-not-rdns": "error",
+        "cid-has-number-prefix": "error",
+        "cid-maybe-not-rdns": "error",
+        "cid-missing-affiliation-gnome": "error",
+        "cid-rnds-contains-hyphen": "error",
```

Comment:
```suggestion
        "cid-rdns-contains-hyphen": "error",
```

Ah

Reply:
This requires at least 1.0.0 https://github.com/ximion/appstream/commit/da92e3c9a9b5c6e9598afedb199744eee1a6a4eb